### PR TITLE
Add page for react-native-tts component

### PR DESCRIPTION
**Pull request by tritao**
2fRBGhy7qF

### CODE REVIEW

File: src/examples/TTSExamplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+'use strict';
+import { Button, FlatList, GestureResponderEvent, StyleSheet, Text, TextInput, View } from 'react-native';
+import React from 'react';
+import Tts, { Voice } from 'react-native-tts';
+import { Example } from '../components/Example';
+import { Page } from '../components/Page';
```
Comment (chiaramooney) [inline src/examples/TTSExamplePage.tsx:6]: nit: Remove extra spaces between text and brackets, make sure you have ESLint turned on in your code editor so files are formatted consistently

File: src/examples/TTSExamplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import Tts from 'react-native-tts';
+  
+function Example() {
+  const onSpeak = () => { Tts.speak("Hello World"); }
+
+  return (
+    <Button onPress={onSpeak} title="Speak"/>
+  );
+}
+  `;
```
Comment (chiaramooney) [inline src/examples/TTSExamplePage.tsx:20]: See comments on #46 about new code snippet content decision

**Review by chiaramooney — COMMENTED** [review]
Make sure ESLint is turned on in your code editor, looks formatting/spacing is off in several places.

File: src/examples/TTSExamplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  const renderVoiceItem = ({ item }: any) => {
+    const voice = item as Voice;
+    return (
+      <View>
+        <Button
+          onPress={(evt) => onVoicePress(evt, voice)}
+          title={voice.name}
+          color={voice.id == selectedVoice?.id ? "#228B22" : "#841584"}
```
Comment (chiaramooney) [inline src/examples/TTSExamplePage.tsx:40]: nit: Could you change the colors here to #E6E6E6, 'red', or 'steelblue' so gallery colors are consistent across pages?

**Review by chiaramooney — APPROVED** [review]
Great!

**Comment by tritao** [comment]
@chiaramooney Rebased again.

**Comment by tritao** [comment]
@chiaramooney Can you take a look at this one?

**Pull request merged**